src/test/kotlin/br/unipar/plano/domain/procedimento/model/factories/ProcedimentoTestConsts.kt

```diff
@@ -0,0 +1,23 @@
+package br.unipar.plano.domain.procedimento.model.factories
+
+import br.unipar.plano.application.factories.CIDADE_TOLEDO_ID
+import br.unipar.plano.domain.centrais.model.IdCentral
+import br.unipar.plano.domain.centrais.model.StatusCentral
+import br.unipar.plano.domain.procedimento.model.Especialidade
+import br.unipar.plano.domain.procedimento.model.IdProcedimento
+import br.unipar.plano.domain.procedimento.model.Servico
+import br.unipar.plano.domain.procedimento.model.StatusProcedimento
+import java.math.BigDecimal
+import java.time.LocalDate
+import java.util.*
+
+val PROCEDIMENTO_CO_ID = IdProcedimento(UUID.fromString("4a2fadf0-4f59-4a2e-b310-a2b264e9e88a"))
```

Comment:
Eu utilizei o CO no meu caso para identificar que a central era a Costa Oeste.

```suggestion
val PROCEDIMENTO_ID = IdProcedimento(UUID.fromString("4a2fadf0-4f59-4a2e-b310-a2b264e9e88a"))
```